layouts/partials/docs/inject/content-before.html

```diff
@@ -0,0 +1,78 @@
+<!-- Site notice -->
+
+<style>
+    #site-notice {
+        position: relative;
+        margin: 2em 0;
+        margin-top: 0;
+        padding: 1em;
+        background-color: #5a83a5;
+        border: 4px solid #335f88;
+        border-radius: 1em;
+    }
+
+    #site-notice * {
+        margin: 0;
+        padding: 0;
+    }
+
+    #site-notice h3 {
+        margin-bottom: 0.5em;
+        font-weight: bold;
+    }
+
+    #site-notice a {
+        color: #f5b642;
+    }
+
+    #site-notice a:hover {
+        color: #ffd78c;
+    }
+
+    #site-notice #notice-close {
+        position: absolute;
+        top: 0;
+        right: 0;
+        visibility: hidden;
+        width: 1em;
+        cursor: pointer;
+        font-size: 1.5em;
+        line-height: 1em;
+    }
+
+    #site-notice:hover #notice-close {
+        visibility: visible;
+    }
+
+    #site-notice #notice-close:hover {
+        color: #c7c7c7;
+    }
+</style>
+
+<div id="site-notice">
+    <h3>NOTICE</h3>
```

Comment:
All-caps is scary, feels like I'm being yelled at

```suggestion
    <h3>New site notice</h3>
```

```suggestion
    <h3>Notice</h3>
```